consent/handler.go

```diff
@@ -133,6 +135,55 @@ func (h *Handler) DeleteUserClientConsentSession(w http.ResponseWriter, r *http.
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// swagger:route GET /oauth2/auth/sessions/consent/{user} oAuth2 listUserClientConsentSessions
```

Comment:
I think the naming should be `listUserConsentSessions`. With the current naming, I would expect the method to be like `listUserClientConsentSessions(user, client string)`